### ISSUE TITLE
beef up Appveyor builds

### ISSUE THREAD

**Issue by slayoo** [tvatter/vinecopulib#44]
Things that are missing:
- building and running the R-dependent tests (needs .lib files for R which are not part of the default distribution)
- compilation for x64 (in addition to i386)
- properly handling Debug and Release modes
- doing "make install" and testing it (this is also not done for Travis actually)

**Comment by slayoo** [comment]
- [ ] set default compiler flags (will fix the unrecognized option '/lpthread' warning)

**Comment by slayoo** [comment]
- [ ] post the built dlls somewhere to the web

**Comment by tvatter** [comment]
Hi,

First, thanks a lot for fixing the compilation problems in appveyor.

Second, since our (Thomas and I) windows skills are extremely limited, I suggest that we postpone all the remaining components of this issue for a later release.

What do you guys think?

**Comment by tnagler** [comment]
Appveyor builds are not really part of the library but rather a tools for development. So we can just release whenever we're ready with the other stuff and keep extending our CI facilities as we go along.

**Comment by slayoo** [comment]
I would suggest just adding execution of the tests before releasing

**Comment by tvatter** [comment]
Yes, this one and maybe eventually the x64 compilation which both should (hopefully) be easy enough.
